app/views/content/teaching-internship-providers/_listing.html.erb

```diff
@@ -27,3 +29,17 @@
      <li>one-to-one guidance from outstanding classroom teachers</li>
      <li>opportunities to network with qualified subject specialists</li>
   </ul>
+
+  <p>“I observed lessons, provided 1:1 support to pupils, and even taught classes! The internship was particularly beneficial in providing a unique insight into teaching training and the application process.” - Carly, former teaching intern. </p>
+
+<h2>How to apply</h2>
+
+<p>You can apply for an internship directly to one of the schools listed below.</p>
+
+<p> To have the best chance of succeeding in your application, <a href="/teacher-training-advisers">get an adviser</a>. They offer one-to-one support, and are all experienced teachers.</p>
```

Comment:
Is this right, to point people to the TTA page rather than the ETA page? Are people doing internships really in their final year?